Tests/bootstrap.php

```diff
@@ -11,18 +11,11 @@
 
 if (file_exists($file = __DIR__.'/../vendor/autoload.php')) {
     $autoload = require_once $file;
+} elseif (file_exists($file = __DIR__.'/../../../autoload.php')) {
+    $autoload = require_once $file;
```

Comment:
This must be removed IMO. Running the testsuite without installing the dependencies in the bundle is not supported (dev dependencies would be missing)